src/libraries/System.Configuration.ConfigurationManager/src/System/Configuration/ApplicationSettingsBase.cs

```diff
@@ -728,15 +728,6 @@ private object GetPropertyValue(string propertyName)
             }
         }
 
-        /// <summary>
-        /// Returns true if this is a clickonce deployed app.
-        /// </summary>
-        internal static bool IsClickOnceDeployed(AppDomain appDomain)
-        {
-            // Always false for .NET Core
-            return false;
-        }
```

Comment:
@ericstj, instead of deleting this, do we need to do anything differently, in light of https://devblogs.microsoft.com/dotnet/announcing-net-5-0-rc-2/#clickonce ?